### ISSUE TITLE
Add test for proportionality assumption

### ISSUE THREAD

**Issue by schuemie** [OHDSI/Cyclops#55]
As part of our suite of study diagnostics, it would be good to evaluate whether the proportionality assumption holds when performing a Cox regression. Could we add something like Schoenfeld residuals to Cyclops?

**Comment by msuchard** [comment]
We may be able to directly use the code from the `survival` package (with an `Rcpp` wrapper from `Cyclops`): https://github.com/therneau/survival/blob/b45e1d20d2ac286424cf68f86a426b55bbb057f2/src/coxscho.c#L36

I'll try to carve out some time to look further.   Do you have a required time-frame in mind?

**Comment by schuemie** [comment]
No, there's no required time-frame. It would just be good to have this eventually.
